apps/portals/src/configurations/elportal/synapseConfigs/projects.ts

```diff
@@ -25,14 +25,14 @@ export const projectCardConfiguration: CardConfiguration = {
     title: 'name',
     subTitle: 'principalInvestigators',
     description: 'abstract',
-    secondaryLabels: ['institutions', 'program', 'grantNumber'],
+    secondaryLabels: ['institutions', 'program', 'grant'],
```

Comment:
Show the 'grant' on a project card: 
![image](https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/17580037/ad1a8025-6c6a-4600-83e0-12d94bccdbcf)